Main latina integration

integrations-ipstudio opened the pull request.

integrations-ipstudio commented:
Hi Awais,

Thanks for merging with the main branch. The next step is to share the page
links with Mariana Tek for QA. Since the project appears to be private,
could you please publish it and provide us with the website link?

Best regards,

*Rahul*
*Integrations*
[image: ipstudio sig block logo-9.png]
ipstudio.co


On Thu, Sep 18, 2025 at 2:07 AM awaisqazi ***@***.***> wrote:

> Merged #1 <https://github.com/awaisqazi/latinasweatproject/pull/1> into
> main.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/awaisqazi/latinasweatproject/pull/1#event-19759367807>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BV3ZBAP6E7Q5RK7LB3T3C333THBBJAVCNFSM6AAAAACGZAN3SWVHI2DSMVQWIX3LMV45UABCJFZXG5LFIV3GK3TUJZXXI2LGNFRWC5DJN5XDWMJZG42TSMZWG44DANY>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

awaisqazi commented:
You should be able to view by adding the page name to the end of the GitHub
preview (ex.
http://awaisqazi.github.io/latinasweatproject/pricing/
)

I can provide all the links later if needed, but that should work for the
time being. I will get the links added to the menu shortly.

Thanks!
Awais Qazi

On Wed, Sep 17, 2025 at 5:03 PM integrations-ipstudio <
***@***.***> wrote:

> *integrations-ipstudio* left a comment (awaisqazi/latinasweatproject#1)
> <https://github.com/awaisqazi/latinasweatproject/pull/1#issuecomment-3304690633>
> Hi Awais,
>
> Thanks for merging with the main branch. The next step is to share the
> page
> links with Mariana Tek for QA. Since the project appears to be private,
> could you please publish it and provide us with the website link?
>
> Best regards,
>
> *Rahul*
> *Integrations*
> [image: ipstudio sig block logo-9.png]
> ipstudio.co
>
>
> On Thu, Sep 18, 2025 at 2:07 AM awaisqazi ***@***.***> wrote:
>
> > Merged #1 <https://github.com/awaisqazi/latinasweatproject/pull/1> into
> > main.
> >
> > —
> > Reply to this email directly, view it on GitHub
> > <
> https://github.com/awaisqazi/latinasweatproject/pull/1#event-19759367807>,
>
> > or unsubscribe
> > <
> https://github.com/notifications/unsubscribe-auth/BV3ZBAP6E7Q5RK7LB3T3C333THBBJAVCNFSM6AAAAACGZAN3SWVHI2DSMVQWIX3LMV45UABCJFZXG5LFIV3GK3TUJZXXI2LGNFRWC5DJN5XDWMJZG42TSMZWG44DANY>
>
> > .
> > You are receiving this because you authored the thread.Message ID:
> > ***@***.***>
> >
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/awaisqazi/latinasweatproject/pull/1#issuecomment-3304690633>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AELHOANWPYQXBNT2QCEJGZD3THLE5AVCNFSM6AAAAACGZAN3SWVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZTGMBUGY4TANRTGM>
> .
> You are receiving this because you modified the open/close state.Message
> ID: ***@***.***>
>

integrations-ipstudio commented:
Hi Awais,

Thanks for sharing the URL, it’s working. However, I had added a password
to the pages using a script — did you remove it? Because the page is no
longer password protected.

Thanks,

Rahul

*Integrations*
[image: ipstudio sig block logo-9.png]
ipstudio.co


On Thu, Sep 18, 2025 at 3:33 AM Integrations IPSTUDIO <
***@***.***> wrote:

> Hi Awais,
>
> Thanks for merging with the main branch. The next step is to share the
> page links with Mariana Tek for QA. Since the project appears to be
> private, could you please publish it and provide us with the website link?
>
> Best regards,
>
> *Rahul*
> *Integrations*
> [image: ipstudio sig block logo-9.png]
> ipstudio.co
>
>
> On Thu, Sep 18, 2025 at 2:07 AM awaisqazi ***@***.***>
> wrote:
>
>> Merged #1 <https://github.com/awaisqazi/latinasweatproject/pull/1> into
>> main.
>>
>> —
>> Reply to this email directly, view it on GitHub
>> <https://github.com/awaisqazi/latinasweatproject/pull/1#event-19759367807>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/BV3ZBAP6E7Q5RK7LB3T3C333THBBJAVCNFSM6AAAAACGZAN3SWVHI2DSMVQWIX3LMV45UABCJFZXG5LFIV3GK3TUJZXXI2LGNFRWC5DJN5XDWMJZG42TSMZWG44DANY>
>> .
>> You are receiving this because you authored the thread.Message ID:
>> ***@***.***>
>>
>

awaisqazi commented:
I don’t believe I changed any of the scripts. Are you able to check the
repo and check that it is still in there?

Let me know, please.

Thank you!


On Wed, Sep 17, 2025 at 5:59 PM integrations-ipstudio <
***@***.***> wrote:

> *integrations-ipstudio* left a comment (awaisqazi/latinasweatproject#1)
> <https://github.com/awaisqazi/latinasweatproject/pull/1#issuecomment-3304790565>
> Hi Awais,
>
> Thanks for sharing the URL, it’s working. However, I had added a password
> to the pages using a script — did you remove it? Because the page is no
> longer password protected.
>
> Thanks,
>
> Rahul
>
> *Integrations*
> [image: ipstudio sig block logo-9.png]
> ipstudio.co
>
>
> On Thu, Sep 18, 2025 at 3:33 AM Integrations IPSTUDIO <
> ***@***.***> wrote:
>
> > Hi Awais,
> >
> > Thanks for merging with the main branch. The next step is to share the
> > page links with Mariana Tek for QA. Since the project appears to be
> > private, could you please publish it and provide us with the website
> link?
> >
> > Best regards,
> >
> > *Rahul*
> > *Integrations*
> > [image: ipstudio sig block logo-9.png]
> > ipstudio.co
> >
> >
> > On Thu, Sep 18, 2025 at 2:07 AM awaisqazi ***@***.***>
> > wrote:
> >
> >> Merged #1 <https://github.com/awaisqazi/latinasweatproject/pull/1>
> into
> >> main.
> >>
> >> —
> >> Reply to this email directly, view it on GitHub
> >> <
> https://github.com/awaisqazi/latinasweatproject/pull/1#event-19759367807>,
>
> >> or unsubscribe
> >> <
> https://github.com/notifications/unsubscribe-auth/BV3ZBAP6E7Q5RK7LB3T3C333THBBJAVCNFSM6AAAAACGZAN3SWVHI2DSMVQWIX3LMV45UABCJFZXG5LFIV3GK3TUJZXXI2LGNFRWC5DJN5XDWMJZG42TSMZWG44DANY>
>
> >> .
> >> You are receiving this because you authored the thread.Message ID:
> >> ***@***.***>
> >>
> >
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/awaisqazi/latinasweatproject/pull/1#issuecomment-3304790565>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AELHOAK4SN66MLRBWPMOPA33THRULAVCNFSM6AAAAACGZAN3SWVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZTGMBUG44TANJWGU>
> .
> You are receiving this because you modified the open/close state.Message
> ID: ***@***.***>
>

integrations-ipstudio commented:
Hi Awais,

Thanks for sharing the URL, it’s working. However, I had added a password
to the pages using a script — did you remove it? Because the page is no
longer password protected.

Thanks,

Rahul
*Integrations*
[image: ipstudio sig block logo-9.png]
ipstudio.co


On Thu, Sep 18, 2025 at 4:28 AM Integrations IPSTUDIO <
***@***.***> wrote:

> Hi Awais,
>
> Thanks for sharing the URL, it’s working. However, I had added a password
> to the pages using a script — did you remove it? Because the page is no
> longer password protected.
>
> Thanks,
>
> Rahul
>
> *Integrations*
> [image: ipstudio sig block logo-9.png]
> ipstudio.co
>
>
> On Thu, Sep 18, 2025 at 3:33 AM Integrations IPSTUDIO <
> ***@***.***> wrote:
>
>> Hi Awais,
>>
>> Thanks for merging with the main branch. The next step is to share the
>> page links with Mariana Tek for QA. Since the project appears to be
>> private, could you please publish it and provide us with the website link?
>>
>> Best regards,
>>
>> *Rahul*
>> *Integrations*
>> [image: ipstudio sig block logo-9.png]
>> ipstudio.co
>>
>>
>> On Thu, Sep 18, 2025 at 2:07 AM awaisqazi ***@***.***>
>> wrote:
>>
>>> Merged #1 <https://github.com/awaisqazi/latinasweatproject/pull/1> into
>>> main.
>>>
>>> —
>>> Reply to this email directly, view it on GitHub
>>> <https://github.com/awaisqazi/latinasweatproject/pull/1#event-19759367807>,
>>> or unsubscribe
>>> <https://github.com/notifications/unsubscribe-auth/BV3ZBAP6E7Q5RK7LB3T3C333THBBJAVCNFSM6AAAAACGZAN3SWVHI2DSMVQWIX3LMV45UABCJFZXG5LFIV3GK3TUJZXXI2LGNFRWC5DJN5XDWMJZG42TSMZWG44DANY>
>>> .
>>> You are receiving this because you authored the thread.Message ID:
>>> ***@***.***>
>>>
>>

integrations-ipstudio commented:
*Hi Awais,*

Okay, I’ll take care of it and let you know once it’s fixed.

Thanks,

Rahul
*Integrations*
[image: ipstudio sig block logo-9.png]
ipstudio.co


On Thu, Sep 18, 2025 at 4:31 AM awaisqazi ***@***.***> wrote:

> *awaisqazi* left a comment (awaisqazi/latinasweatproject#1)
> <https://github.com/awaisqazi/latinasweatproject/pull/1#issuecomment-3304793578>
> I don’t believe I changed any of the scripts. Are you able to check the
> repo and check that it is still in there?
>
> Let me know, please.
>
> Thank you!
>
>
> On Wed, Sep 17, 2025 at 5:59 PM integrations-ipstudio <
> ***@***.***> wrote:
>
> > *integrations-ipstudio* left a comment (awaisqazi/latinasweatproject#1)
> > <
> https://github.com/awaisqazi/latinasweatproject/pull/1#issuecomment-3304790565>
>
> > Hi Awais,
> >
> > Thanks for sharing the URL, it’s working. However, I had added a
> password
> > to the pages using a script — did you remove it? Because the page is no
> > longer password protected.
> >
> > Thanks,
> >
> > Rahul
> >
> > *Integrations*
> > [image: ipstudio sig block logo-9.png]
> > ipstudio.co
> >
> >
> > On Thu, Sep 18, 2025 at 3:33 AM Integrations IPSTUDIO <
> > ***@***.***> wrote:
> >
> > > Hi Awais,
> > >
> > > Thanks for merging with the main branch. The next step is to share the
> > > page links with Mariana Tek for QA. Since the project appears to be
> > > private, could you please publish it and provide us with the website
> > link?
> > >
> > > Best regards,
> > >
> > > *Rahul*
> > > *Integrations*
> > > [image: ipstudio sig block logo-9.png]
> > > ipstudio.co
> > >
> > >
> > > On Thu, Sep 18, 2025 at 2:07 AM awaisqazi ***@***.***>
> > > wrote:
> > >
> > >> Merged #1 <https://github.com/awaisqazi/latinasweatproject/pull/1>
> > into
> > >> main.
> > >>
> > >> —
> > >> Reply to this email directly, view it on GitHub
> > >> <
> > https://github.com/awaisqazi/latinasweatproject/pull/1#event-19759367807>,
>
> >
> > >> or unsubscribe
> > >> <
> >
> https://github.com/notifications/unsubscribe-auth/BV3ZBAP6E7Q5RK7LB3T3C333THBBJAVCNFSM6AAAAACGZAN3SWVHI2DSMVQWIX3LMV45UABCJFZXG5LFIV3GK3TUJZXXI2LGNFRWC5DJN5XDWMJZG42TSMZWG44DANY>
>
> >
> > >> .
> > >> You are receiving this because you authored the thread.Message ID:
> > >> ***@***.***>
> > >>
> > >
> >
> > —
> > Reply to this email directly, view it on GitHub
> > <
> https://github.com/awaisqazi/latinasweatproject/pull/1#issuecomment-3304790565>,
>
> > or unsubscribe
> > <
> https://github.com/notifications/unsubscribe-auth/AELHOAK4SN66MLRBWPMOPA33THRULAVCNFSM6AAAAACGZAN3SWVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZTGMBUG44TANJWGU>
>
> > .
> > You are receiving this because you modified the open/close state.Message
> > ID: ***@***.***>
> >
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/awaisqazi/latinasweatproject/pull/1#issuecomment-3304793578>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BV3ZBAPYV4XWOZUUGYO3ZTD3THR3PAVCNFSM6AAAAACGZAN3SWVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZTGMBUG44TGNJXHA>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

integrations-ipstudio commented:
Hi Awais,

I noticed that the password protection code was commented out. I’ve
uncommented it, made a small path-related change, and created a pull
request under the branch Rahul-max55:main-password-protected.

Please go ahead and merge this branch. After merging, the code should run
as expected. We can review again if any issues come up.

Thanks,

Rahul
*Integrations*
[image: ipstudio sig block logo-9.png]
ipstudio.co


On Thu, Sep 18, 2025 at 5:20 AM Integrations IPSTUDIO <
***@***.***> wrote:

> *Hi Awais,*
>
> Okay, I’ll take care of it and let you know once it’s fixed.
>
> Thanks,
>
> Rahul
> *Integrations*
> [image: ipstudio sig block logo-9.png]
> ipstudio.co
>
>
> On Thu, Sep 18, 2025 at 4:31 AM awaisqazi ***@***.***>
> wrote:
>
>> *awaisqazi* left a comment (awaisqazi/latinasweatproject#1)
>> <https://github.com/awaisqazi/latinasweatproject/pull/1#issuecomment-3304793578>
>> I don’t believe I changed any of the scripts. Are you able to check the
>> repo and check that it is still in there?
>>
>> Let me know, please.
>>
>> Thank you!
>>
>>
>> On Wed, Sep 17, 2025 at 5:59 PM integrations-ipstudio <
>> ***@***.***> wrote:
>>
>> > *integrations-ipstudio* left a comment (awaisqazi/latinasweatproject#1)
>> > <
>> https://github.com/awaisqazi/latinasweatproject/pull/1#issuecomment-3304790565>
>>
>> > Hi Awais,
>> >
>> > Thanks for sharing the URL, it’s working. However, I had added a
>> password
>> > to the pages using a script — did you remove it? Because the page is no
>> > longer password protected.
>> >
>> > Thanks,
>> >
>> > Rahul
>> >
>> > *Integrations*
>> > [image: ipstudio sig block logo-9.png]
>> > ipstudio.co
>> >
>> >
>> > On Thu, Sep 18, 2025 at 3:33 AM Integrations IPSTUDIO <
>> > ***@***.***> wrote:
>> >
>> > > Hi Awais,
>> > >
>> > > Thanks for merging with the main branch. The next step is to share
>> the
>> > > page links with Mariana Tek for QA. Since the project appears to be
>> > > private, could you please publish it and provide us with the website
>> > link?
>> > >
>> > > Best regards,
>> > >
>> > > *Rahul*
>> > > *Integrations*
>> > > [image: ipstudio sig block logo-9.png]
>> > > ipstudio.co
>> > >
>> > >
>> > > On Thu, Sep 18, 2025 at 2:07 AM awaisqazi ***@***.***>
>> > > wrote:
>> > >
>> > >> Merged #1 <https://github.com/awaisqazi/latinasweatproject/pull/1>
>> > into
>> > >> main.
>> > >>
>> > >> —
>> > >> Reply to this email directly, view it on GitHub
>> > >> <
>> >
>> https://github.com/awaisqazi/latinasweatproject/pull/1#event-19759367807>,
>>
>> >
>> > >> or unsubscribe
>> > >> <
>> >
>> https://github.com/notifications/unsubscribe-auth/BV3ZBAP6E7Q5RK7LB3T3C333THBBJAVCNFSM6AAAAACGZAN3SWVHI2DSMVQWIX3LMV45UABCJFZXG5LFIV3GK3TUJZXXI2LGNFRWC5DJN5XDWMJZG42TSMZWG44DANY>
>>
>> >
>> > >> .
>> > >> You are receiving this because you authored the thread.Message ID:
>> > >> ***@***.***>
>> > >>
>> > >
>> >
>> > —
>> > Reply to this email directly, view it on GitHub
>> > <
>> https://github.com/awaisqazi/latinasweatproject/pull/1#issuecomment-3304790565>,
>>
>> > or unsubscribe
>> > <
>> https://github.com/notifications/unsubscribe-auth/AELHOAK4SN66MLRBWPMOPA33THRULAVCNFSM6AAAAACGZAN3SWVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZTGMBUG44TANJWGU>
>>
>> > .
>> > You are receiving this because you modified the open/close
>> state.Message
>> > ID: ***@***.***>
>> >
>>
>> —
>> Reply to this email directly, view it on GitHub
>> <https://github.com/awaisqazi/latinasweatproject/pull/1#issuecomment-3304793578>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/BV3ZBAPYV4XWOZUUGYO3ZTD3THR3PAVCNFSM6AAAAACGZAN3SWVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZTGMBUG44TGNJXHA>
>> .
>> You are receiving this because you authored the thread.Message ID:
>> ***@***.***>
>>
>

awaisqazi commented:
Why are we password protecting those pages? As we are gearing up for
launch, we are planning on adding in our schedule and making that visible.
We should be okay to remove the password protection from those pages,
right?

Let me know, please. Thank you!

Sincerely,
Fez

On Thu, Sep 18, 2025 at 12:12 PM integrations-ipstudio <
***@***.***> wrote:

> *integrations-ipstudio* left a comment (awaisqazi/latinasweatproject#1)
> <https://github.com/awaisqazi/latinasweatproject/pull/1#issuecomment-3308622726>
> Hi Awais,
>
> I noticed that the password protection code was commented out. I’ve
> uncommented it, made a small path-related change, and created a pull
> request under the branch Rahul-max55:main-password-protected.
>
> Please go ahead and merge this branch. After merging, the code should run
> as expected. We can review again if any issues come up.
>
> Thanks,
>
> Rahul
> *Integrations*
> [image: ipstudio sig block logo-9.png]
> ipstudio.co
>
>
> On Thu, Sep 18, 2025 at 5:20 AM Integrations IPSTUDIO <
> ***@***.***> wrote:
>
> > *Hi Awais,*
> >
> > Okay, I’ll take care of it and let you know once it’s fixed.
> >
> > Thanks,
> >
> > Rahul
> > *Integrations*
> > [image: ipstudio sig block logo-9.png]
> > ipstudio.co
> >
> >
> > On Thu, Sep 18, 2025 at 4:31 AM awaisqazi ***@***.***>
> > wrote:
> >
> >> *awaisqazi* left a comment (awaisqazi/latinasweatproject#1)
> >> <
> https://github.com/awaisqazi/latinasweatproject/pull/1#issuecomment-3304793578>
>
> >> I don’t believe I changed any of the scripts. Are you able to check the
> >> repo and check that it is still in there?
> >>
> >> Let me know, please.
> >>
> >> Thank you!
> >>
> >>
> >> On Wed, Sep 17, 2025 at 5:59 PM integrations-ipstudio <
> >> ***@***.***> wrote:
> >>
> >> > *integrations-ipstudio* left a comment
> (awaisqazi/latinasweatproject#1)
> >> > <
> >>
> https://github.com/awaisqazi/latinasweatproject/pull/1#issuecomment-3304790565>
>
> >>
> >> > Hi Awais,
> >> >
> >> > Thanks for sharing the URL, it’s working. However, I had added a
> >> password
> >> > to the pages using a script — did you remove it? Because the page is
> no
> >> > longer password protected.
> >> >
> >> > Thanks,
> >> >
> >> > Rahul
> >> >
> >> > *Integrations*
> >> > [image: ipstudio sig block logo-9.png]
> >> > ipstudio.co
> >> >
> >> >
> >> > On Thu, Sep 18, 2025 at 3:33 AM Integrations IPSTUDIO <
> >> > ***@***.***> wrote:
> >> >
> >> > > Hi Awais,
> >> > >
> >> > > Thanks for merging with the main branch. The next step is to share
> >> the
> >> > > page links with Mariana Tek for QA. Since the project appears to be
> >> > > private, could you please publish it and provide us with the
> website
> >> > link?
> >> > >
> >> > > Best regards,
> >> > >
> >> > > *Rahul*
> >> > > *Integrations*
> >> > > [image: ipstudio sig block logo-9.png]
> >> > > ipstudio.co
> >> > >
> >> > >
> >> > > On Thu, Sep 18, 2025 at 2:07 AM awaisqazi ***@***.***>
> >> > > wrote:
> >> > >
> >> > >> Merged #1 <https://github.com/awaisqazi/latinasweatproject/pull/1>
>
> >> > into
> >> > >> main.
> >> > >>
> >> > >> —
> >> > >> Reply to this email directly, view it on GitHub
> >> > >> <
> >> >
> >>
> https://github.com/awaisqazi/latinasweatproject/pull/1#event-19759367807>,
>
> >>
> >> >
> >> > >> or unsubscribe
> >> > >> <
> >> >
> >>
> https://github.com/notifications/unsubscribe-auth/BV3ZBAP6E7Q5RK7LB3T3C333THBBJAVCNFSM6AAAAACGZAN3SWVHI2DSMVQWIX3LMV45UABCJFZXG5LFIV3GK3TUJZXXI2LGNFRWC5DJN5XDWMJZG42TSMZWG44DANY>
>
> >>
> >> >
> >> > >> .
> >> > >> You are receiving this because you authored the thread.Message ID:
> >> > >> ***@***.***>
> >> > >>
> >> > >
> >> >
> >> > —
> >> > Reply to this email directly, view it on GitHub
> >> > <
> >>
> https://github.com/awaisqazi/latinasweatproject/pull/1#issuecomment-3304790565>,
>
> >>
> >> > or unsubscribe
> >> > <
> >>
> https://github.com/notifications/unsubscribe-auth/AELHOAK4SN66MLRBWPMOPA33THRULAVCNFSM6AAAAACGZAN3SWVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZTGMBUG44TANJWGU>
>
> >>
> >> > .
> >> > You are receiving this because you modified the open/close
> >> state.Message
> >> > ID: ***@***.***>
> >> >
> >>
> >> —
> >> Reply to this email directly, view it on GitHub
> >> <
> https://github.com/awaisqazi/latinasweatproject/pull/1#issuecomment-3304793578>,
>
> >> or unsubscribe
> >> <
> https://github.com/notifications/unsubscribe-auth/BV3ZBAPYV4XWOZUUGYO3ZTD3THR3PAVCNFSM6AAAAACGZAN3SWVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZTGMBUG44TGNJXHA>
>
> >> .
> >> You are receiving this because you authored the thread.Message ID:
> >> ***@***.***>
> >>
> >
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/awaisqazi/latinasweatproject/pull/1#issuecomment-3308622726>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AELHOAMJYGK2REBLWICFBPL3TLRWDAVCNFSM6AAAAACGZAN3SWVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZTGMBYGYZDENZSGY>
> .
> You are receiving this because you modified the open/close state.Message
> ID: ***@***.***>
>

integrations-ipstudio commented:
Hi Fez,

During the web integration staging phase, we keep those pages password
protected. At the time of launch, we will remove the password protection.

If you’d like us to make the pages accessible without a password now so the
team can review, we can certainly do that. Please confirm if you’d like me
to proceed this way, and I’ll follow the process accordingly.

Best regards,

Rahul
*Integrations*
[image: ipstudio sig block logo-9.png]
ipstudio.co


On Fri, Sep 19, 2025 at 2:43 AM awaisqazi ***@***.***> wrote:

> *awaisqazi* left a comment (awaisqazi/latinasweatproject#1)
> <https://github.com/awaisqazi/latinasweatproject/pull/1#issuecomment-3309744386>
> Why are we password protecting those pages? As we are gearing up for
> launch, we are planning on adding in our schedule and making that visible.
> We should be okay to remove the password protection from those pages,
> right?
>
> Let me know, please. Thank you!
>
> Sincerely,
> Fez
>
> On Thu, Sep 18, 2025 at 12:12 PM integrations-ipstudio <
> ***@***.***> wrote:
>
> > *integrations-ipstudio* left a comment (awaisqazi/latinasweatproject#1)
> > <
> https://github.com/awaisqazi/latinasweatproject/pull/1#issuecomment-3308622726>
>
> > Hi Awais,
> >
> > I noticed that the password protection code was commented out. I’ve
> > uncommented it, made a small path-related change, and created a pull
> > request under the branch Rahul-max55:main-password-protected.
> >
> > Please go ahead and merge this branch. After merging, the code should
> run
> > as expected. We can review again if any issues come up.
> >
> > Thanks,
> >
> > Rahul
> > *Integrations*
> > [image: ipstudio sig block logo-9.png]
> > ipstudio.co
> >
> >
> > On Thu, Sep 18, 2025 at 5:20 AM Integrations IPSTUDIO <
> > ***@***.***> wrote:
> >
> > > *Hi Awais,*
> > >
> > > Okay, I’ll take care of it and let you know once it’s fixed.
> > >
> > > Thanks,
> > >
> > > Rahul
> > > *Integrations*
> > > [image: ipstudio sig block logo-9.png]
> > > ipstudio.co
> > >
> > >
> > > On Thu, Sep 18, 2025 at 4:31 AM awaisqazi ***@***.***>
> > > wrote:
> > >
> > >> *awaisqazi* left a comment (awaisqazi/latinasweatproject#1)
> > >> <
> >
> https://github.com/awaisqazi/latinasweatproject/pull/1#issuecomment-3304793578>
>
> >
> > >> I don’t believe I changed any of the scripts. Are you able to check
> the
> > >> repo and check that it is still in there?
> > >>
> > >> Let me know, please.
> > >>
> > >> Thank you!
> > >>
> > >>
> > >> On Wed, Sep 17, 2025 at 5:59 PM integrations-ipstudio <
> > >> ***@***.***> wrote:
> > >>
> > >> > *integrations-ipstudio* left a comment
> > (awaisqazi/latinasweatproject#1)
> > >> > <
> > >>
> >
> https://github.com/awaisqazi/latinasweatproject/pull/1#issuecomment-3304790565>
>
> >
> > >>
> > >> > Hi Awais,
> > >> >
> > >> > Thanks for sharing the URL, it’s working. However, I had added a
> > >> password
> > >> > to the pages using a script — did you remove it? Because the page
> is
> > no
> > >> > longer password protected.
> > >> >
> > >> > Thanks,
> > >> >
> > >> > Rahul
> > >> >
> > >> > *Integrations*
> > >> > [image: ipstudio sig block logo-9.png]
> > >> > ipstudio.co
> > >> >
> > >> >
> > >> > On Thu, Sep 18, 2025 at 3:33 AM Integrations IPSTUDIO <
> > >> > ***@***.***> wrote:
> > >> >
> > >> > > Hi Awais,
> > >> > >
> > >> > > Thanks for merging with the main branch. The next step is to
> share
> > >> the
> > >> > > page links with Mariana Tek for QA. Since the project appears to
> be
> > >> > > private, could you please publish it and provide us with the
> > website
> > >> > link?
> > >> > >
> > >> > > Best regards,
> > >> > >
> > >> > > *Rahul*
> > >> > > *Integrations*
> > >> > > [image: ipstudio sig block logo-9.png]
> > >> > > ipstudio.co
> > >> > >
> > >> > >
> > >> > > On Thu, Sep 18, 2025 at 2:07 AM awaisqazi ***@***.***>
> > >> > > wrote:
> > >> > >
> > >> > >> Merged #1 <
> https://github.com/awaisqazi/latinasweatproject/pull/1>
> >
> > >> > into
> > >> > >> main.
> > >> > >>
> > >> > >> —
> > >> > >> Reply to this email directly, view it on GitHub
> > >> > >> <
> > >> >
> > >>
> > https://github.com/awaisqazi/latinasweatproject/pull/1#event-19759367807>,
>
> >
> > >>
> > >> >
> > >> > >> or unsubscribe
> > >> > >> <
> > >> >
> > >>
> >
> https://github.com/notifications/unsubscribe-auth/BV3ZBAP6E7Q5RK7LB3T3C333THBBJAVCNFSM6AAAAACGZAN3SWVHI2DSMVQWIX3LMV45UABCJFZXG5LFIV3GK3TUJZXXI2LGNFRWC5DJN5XDWMJZG42TSMZWG44DANY>
>
> >
> > >>
> > >> >
> > >> > >> .
> > >> > >> You are receiving this because you authored the thread.Message
> ID:
> > >> > >> ***@***.***>
> > >> > >>
> > >> > >
> > >> >
> > >> > —
> > >> > Reply to this email directly, view it on GitHub
> > >> > <
> > >>
> >
> https://github.com/awaisqazi/latinasweatproject/pull/1#issuecomment-3304790565>,
>
> >
> > >>
> > >> > or unsubscribe
> > >> > <
> > >>
> >
> https://github.com/notifications/unsubscribe-auth/AELHOAK4SN66MLRBWPMOPA33THRULAVCNFSM6AAAAACGZAN3SWVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZTGMBUG44TANJWGU>
>
> >
> > >>
> > >> > .
> > >> > You are receiving this because you modified the open/close
> > >> state.Message
> > >> > ID: ***@***.***>
> > >> >
> > >>
> > >> —
> > >> Reply to this email directly, view it on GitHub
> > >> <
> >
> https://github.com/awaisqazi/latinasweatproject/pull/1#issuecomment-3304793578>,
>
> >
> > >> or unsubscribe
> > >> <
> >
> https://github.com/notifications/unsubscribe-auth/BV3ZBAPYV4XWOZUUGYO3ZTD3THR3PAVCNFSM6AAAAACGZAN3SWVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZTGMBUG44TGNJXHA>
>
> >
> > >> .
> > >> You are receiving this because you authored the thread.Message ID:
> > >> ***@***.***>
> > >>
> > >
> >
> > —
> > Reply to this email directly, view it on GitHub
> > <
> https://github.com/awaisqazi/latinasweatproject/pull/1#issuecomment-3308622726>,
>
> > or unsubscribe
> > <
> https://github.com/notifications/unsubscribe-auth/AELHOAMJYGK2REBLWICFBPL3TLRWDAVCNFSM6AAAAACGZAN3SWVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZTGMBYGYZDENZSGY>
>
> > .
> > You are receiving this because you modified the open/close state.Message
> > ID: ***@***.***>
> >
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/awaisqazi/latinasweatproject/pull/1#issuecomment-3309744386>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BV3ZBANETNJWGBAINUXBEAT3TMN6TAVCNFSM6AAAAACGZAN3SWVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZTGMBZG42DIMZYGY>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

awaisqazi commented:
Yes, please! Also, wondering if it is possible for you to make the "My
Account" button in the header dynamic so that when user is logged in, their
photo appears next to the "My Account" link? Is that something that is
easily accomplishable? If not, no worries.

Thank you!

Sincerely,
Fez

On Thu, Sep 18, 2025 at 4:21 PM integrations-ipstudio <
***@***.***> wrote:

> *integrations-ipstudio* left a comment (awaisqazi/latinasweatproject#1)
> <https://github.com/awaisqazi/latinasweatproject/pull/1#issuecomment-3309763966>
> Hi Fez,
>
> During the web integration staging phase, we keep those pages password
> protected. At the time of launch, we will remove the password protection.
>
> If you’d like us to make the pages accessible without a password now so
> the
> team can review, we can certainly do that. Please confirm if you’d like me
> to proceed this way, and I’ll follow the process accordingly.
>
> Best regards,
>
> Rahul
> *Integrations*
> [image: ipstudio sig block logo-9.png]
> ipstudio.co
>
>
> On Fri, Sep 19, 2025 at 2:43 AM awaisqazi ***@***.***> wrote:
>
> > *awaisqazi* left a comment (awaisqazi/latinasweatproject#1)
> > <
> https://github.com/awaisqazi/latinasweatproject/pull/1#issuecomment-3309744386>
>
> > Why are we password protecting those pages? As we are gearing up for
> > launch, we are planning on adding in our schedule and making that
> visible.
> > We should be okay to remove the password protection from those pages,
> > right?
> >
> > Let me know, please. Thank you!
> >
> > Sincerely,
> > Fez
> >
> > On Thu, Sep 18, 2025 at 12:12 PM integrations-ipstudio <
> > ***@***.***> wrote:
> >
> > > *integrations-ipstudio* left a comment
> (awaisqazi/latinasweatproject#1)
> > > <
> >
> https://github.com/awaisqazi/latinasweatproject/pull/1#issuecomment-3308622726>
>
> >
> > > Hi Awais,
> > >
> > > I noticed that the password protection code was commented out. I’ve
> > > uncommented it, made a small path-related change, and created a pull
> > > request under the branch Rahul-max55:main-password-protected.
> > >
> > > Please go ahead and merge this branch. After merging, the code should
> > run
> > > as expected. We can review again if any issues come up.
> > >
> > > Thanks,
> > >
> > > Rahul
> > > *Integrations*
> > > [image: ipstudio sig block logo-9.png]
> > > ipstudio.co
> > >
> > >
> > > On Thu, Sep 18, 2025 at 5:20 AM Integrations IPSTUDIO <
> > > ***@***.***> wrote:
> > >
> > > > *Hi Awais,*
> > > >
> > > > Okay, I’ll take care of it and let you know once it’s fixed.
> > > >
> > > > Thanks,
> > > >
> > > > Rahul
> > > > *Integrations*
> > > > [image: ipstudio sig block logo-9.png]
> > > > ipstudio.co
> > > >
> > > >
> > > > On Thu, Sep 18, 2025 at 4:31 AM awaisqazi ***@***.***>
> > > > wrote:
> > > >
> > > >> *awaisqazi* left a comment (awaisqazi/latinasweatproject#1)
> > > >> <
> > >
> >
> https://github.com/awaisqazi/latinasweatproject/pull/1#issuecomment-3304793578>
>
> >
> > >
> > > >> I don’t believe I changed any of the scripts. Are you able to check
> > the
> > > >> repo and check that it is still in there?
> > > >>
> > > >> Let me know, please.
> > > >>
> > > >> Thank you!
> > > >>
> > > >>
> > > >> On Wed, Sep 17, 2025 at 5:59 PM integrations-ipstudio <
> > > >> ***@***.***> wrote:
> > > >>
> > > >> > *integrations-ipstudio* left a comment
> > > (awaisqazi/latinasweatproject#1)
> > > >> > <
> > > >>
> > >
> >
> https://github.com/awaisqazi/latinasweatproject/pull/1#issuecomment-3304790565>
>
> >
> > >
> > > >>
> > > >> > Hi Awais,
> > > >> >
> > > >> > Thanks for sharing the URL, it’s working. However, I had added a
> > > >> password
> > > >> > to the pages using a script — did you remove it? Because the page
> > is
> > > no
> > > >> > longer password protected.
> > > >> >
> > > >> > Thanks,
> > > >> >
> > > >> > Rahul
> > > >> >
> > > >> > *Integrations*
> > > >> > [image: ipstudio sig block logo-9.png]
> > > >> > ipstudio.co
> > > >> >
> > > >> >
> > > >> > On Thu, Sep 18, 2025 at 3:33 AM Integrations IPSTUDIO <
> > > >> > ***@***.***> wrote:
> > > >> >
> > > >> > > Hi Awais,
> > > >> > >
> > > >> > > Thanks for merging with the main branch. The next step is to
> > share
> > > >> the
> > > >> > > page links with Mariana Tek for QA. Since the project appears
> to
> > be
> > > >> > > private, could you please publish it and provide us with the
> > > website
> > > >> > link?
> > > >> > >
> > > >> > > Best regards,
> > > >> > >
> > > >> > > *Rahul*
> > > >> > > *Integrations*
> > > >> > > [image: ipstudio sig block logo-9.png]
> > > >> > > ipstudio.co
> > > >> > >
> > > >> > >
> > > >> > > On Thu, Sep 18, 2025 at 2:07 AM awaisqazi ***@***.***>
> > > >> > > wrote:
> > > >> > >
> > > >> > >> Merged #1 <
> > https://github.com/awaisqazi/latinasweatproject/pull/1>
> > >
> > > >> > into
> > > >> > >> main.
> > > >> > >>
> > > >> > >> —
> > > >> > >> Reply to this email directly, view it on GitHub
> > > >> > >> <
> > > >> >
> > > >>
> > >
> https://github.com/awaisqazi/latinasweatproject/pull/1#event-19759367807>,
>
> >
> > >
> > > >>
> > > >> >
> > > >> > >> or unsubscribe
> > > >> > >> <
> > > >> >
> > > >>
> > >
> >
> https://github.com/notifications/unsubscribe-auth/BV3ZBAP6E7Q5RK7LB3T3C333THBBJAVCNFSM6AAAAACGZAN3SWVHI2DSMVQWIX3LMV45UABCJFZXG5LFIV3GK3TUJZXXI2LGNFRWC5DJN5XDWMJZG42TSMZWG44DANY>
>
> >
> > >
> > > >>
> > > >> >
> > > >> > >> .
> > > >> > >> You are receiving this because you authored the thread.Message
> > ID:
> > > >> > >> ***@***.***>
> > > >> > >>
> > > >> > >
> > > >> >
> > > >> > —
> > > >> > Reply to this email directly, view it on GitHub
> > > >> > <
> > > >>
> > >
> >
> https://github.com/awaisqazi/latinasweatproject/pull/1#issuecomment-3304790565>,
>
> >
> > >
> > > >>
> > > >> > or unsubscribe
> > > >> > <
> > > >>
> > >
> >
> https://github.com/notifications/unsubscribe-auth/AELHOAK4SN66MLRBWPMOPA33THRULAVCNFSM6AAAAACGZAN3SWVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZTGMBUG44TANJWGU>
>
> >
> > >
> > > >>
> > > >> > .
> > > >> > You are receiving this because you modified the open/close
> > > >> state.Message
> > > >> > ID: ***@***.***>
> > > >> >
> > > >>
> > > >> —
> > > >> Reply to this email directly, view it on GitHub
> > > >> <
> > >
> >
> https://github.com/awaisqazi/latinasweatproject/pull/1#issuecomment-3304793578>,
>
> >
> > >
> > > >> or unsubscribe
> > > >> <
> > >
> >
> https://github.com/notifications/unsubscribe-auth/BV3ZBAPYV4XWOZUUGYO3ZTD3THR3PAVCNFSM6AAAAACGZAN3SWVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZTGMBUG44TGNJXHA>
>
> >
> > >
> > > >> .
> > > >> You are receiving this because you authored the thread.Message ID:
> > > >> ***@***.***>
> > > >>
> > > >
> > >
> > > —
> > > Reply to this email directly, view it on GitHub
> > > <
> >
> https://github.com/awaisqazi/latinasweatproject/pull/1#issuecomment-3308622726>,
>
> >
> > > or unsubscribe
> > > <
> >
> https://github.com/notifications/unsubscribe-auth/AELHOAMJYGK2REBLWICFBPL3TLRWDAVCNFSM6AAAAACGZAN3SWVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZTGMBYGYZDENZSGY>
>
> >
> > > .
> > > You are receiving this because you modified the open/close
> state.Message
> > > ID: ***@***.***>
> > >
> >
> > —
> > Reply to this email directly, view it on GitHub
> > <
> https://github.com/awaisqazi/latinasweatproject/pull/1#issuecomment-3309744386>,
>
> > or unsubscribe
> > <
> https://github.com/notifications/unsubscribe-auth/BV3ZBANETNJWGBAINUXBEAT3TMN6TAVCNFSM6AAAAACGZAN3SWVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZTGMBZG42DIMZYGY>
>
> > .
> > You are receiving this because you authored the thread.Message ID:
> > ***@***.***>
> >
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/awaisqazi/latinasweatproject/pull/1#issuecomment-3309763966>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AELHOALI7Z2JCYDIIVEVUSD3TMO6BAVCNFSM6AAAAACGZAN3SWVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZTGMBZG43DGOJWGY>
> .
> You are receiving this because you modified the open/close state.Message
> ID: ***@***.***>
>

integrations-ipstudio commented:
Hi Fez,

That’s a great idea! However, due to the limitations of our web
integration, we’re not able to make the “My Account” button dynamic or
display the user’s photo in the header. Sorry about this — it’s outside the
scope of what we can implement through integration. I’ll share the web
integration pages links with Mariana Tek

Thanks,

Rahul
*Integrations*
[image: ipstudio sig block logo-9.png]
ipstudio.co


On Fri, Sep 19, 2025 at 3:01 AM awaisqazi ***@***.***> wrote:

> *awaisqazi* left a comment (awaisqazi/latinasweatproject#1)
> <https://github.com/awaisqazi/latinasweatproject/pull/1#issuecomment-3309785190>
> Yes, please! Also, wondering if it is possible for you to make the "My
> Account" button in the header dynamic so that when user is logged in,
> their
> photo appears next to the "My Account" link? Is that something that is
> easily accomplishable? If not, no worries.
>
> Thank you!
>
> Sincerely,
> Fez
>
> On Thu, Sep 18, 2025 at 4:21 PM integrations-ipstudio <
> ***@***.***> wrote:
>
> > *integrations-ipstudio* left a comment (awaisqazi/latinasweatproject#1)
> > <
> https://github.com/awaisqazi/latinasweatproject/pull/1#issuecomment-3309763966>
>
> > Hi Fez,
> >
> > During the web integration staging phase, we keep those pages password
> > protected. At the time of launch, we will remove the password
> protection.
> >
> > If you’d like us to make the pages accessible without a password now so
> > the
> > team can review, we can certainly do that. Please confirm if you’d like
> me
> > to proceed this way, and I’ll follow the process accordingly.
> >
> > Best regards,
> >
> > Rahul
> > *Integrations*
> > [image: ipstudio sig block logo-9.png]
> > ipstudio.co
> >
> >
> > On Fri, Sep 19, 2025 at 2:43 AM awaisqazi ***@***.***> wrote:
> >
> > > *awaisqazi* left a comment (awaisqazi/latinasweatproject#1)
> > > <
> >
> https://github.com/awaisqazi/latinasweatproject/pull/1#issuecomment-3309744386>
>
> >
> > > Why are we password protecting those pages? As we are gearing up for
> > > launch, we are planning on adding in our schedule and making that
> > visible.
> > > We should be okay to remove the password protection from those pages,
> > > right?
> > >
> > > Let me know, please. Thank you!
> > >
> > > Sincerely,
> > > Fez
> > >
> > > On Thu, Sep 18, 2025 at 12:12 PM integrations-ipstudio <
> > > ***@***.***> wrote:
> > >
> > > > *integrations-ipstudio* left a comment
> > (awaisqazi/latinasweatproject#1)
> > > > <
> > >
> >
> https://github.com/awaisqazi/latinasweatproject/pull/1#issuecomment-3308622726>
>
> >
> > >
> > > > Hi Awais,
> > > >
> > > > I noticed that the password protection code was commented out. I’ve
> > > > uncommented it, made a small path-related change, and created a pull
> > > > request under the branch Rahul-max55:main-password-protected.
> > > >
> > > > Please go ahead and merge this branch. After merging, the code
> should
> > > run
> > > > as expected. We can review again if any issues come up.
> > > >
> > > > Thanks,
> > > >
> > > > Rahul
> > > > *Integrations*
> > > > [image: ipstudio sig block logo-9.png]
> > > > ipstudio.co
> > > >
> > > >
> > > > On Thu, Sep 18, 2025 at 5:20 AM Integrations IPSTUDIO <
> > > > ***@***.***> wrote:
> > > >
> > > > > *Hi Awais,*
> > > > >
> > > > > Okay, I’ll take care of it and let you know once it’s fixed.
> > > > >
> > > > > Thanks,
> > > > >
> > > > > Rahul
> > > > > *Integrations*
> > > > > [image: ipstudio sig block logo-9.png]
> > > > > ipstudio.co
> > > > >
> > > > >
> > > > > On Thu, Sep 18, 2025 at 4:31 AM awaisqazi ***@***.***>
> > > > > wrote:
> > > > >
> > > > >> *awaisqazi* left a comment (awaisqazi/latinasweatproject#1)
> > > > >> <
> > > >
> > >
> >
> https://github.com/awaisqazi/latinasweatproject/pull/1#issuecomment-3304793578>
>
> >
> > >
> > > >
> > > > >> I don’t believe I changed any of the scripts. Are you able to
> check
> > > the
> > > > >> repo and check that it is still in there?
> > > > >>
> > > > >> Let me know, please.
> > > > >>
> > > > >> Thank you!
> > > > >>
> > > > >>
> > > > >> On Wed, Sep 17, 2025 at 5:59 PM integrations-ipstudio <
> > > > >> ***@***.***> wrote:
> > > > >>
> > > > >> > *integrations-ipstudio* left a comment
> > > > (awaisqazi/latinasweatproject#1)
> > > > >> > <
> > > > >>
> > > >
> > >
> >
> https://github.com/awaisqazi/latinasweatproject/pull/1#issuecomment-3304790565>
>
> >
> > >
> > > >
> > > > >>
> > > > >> > Hi Awais,
> > > > >> >
> > > > >> > Thanks for sharing the URL, it’s working. However, I had added
> a
> > > > >> password
> > > > >> > to the pages using a script — did you remove it? Because the
> page
> > > is
> > > > no
> > > > >> > longer password protected.
> > > > >> >
> > > > >> > Thanks,
> > > > >> >
> > > > >> > Rahul
> > > > >> >
> > > > >> > *Integrations*
> > > > >> > [image: ipstudio sig block logo-9.png]
> > > > >> > ipstudio.co
> > > > >> >
> > > > >> >
> > > > >> > On Thu, Sep 18, 2025 at 3:33 AM Integrations IPSTUDIO <
> > > > >> > ***@***.***> wrote:
> > > > >> >
> > > > >> > > Hi Awais,
> > > > >> > >
> > > > >> > > Thanks for merging with the main branch. The next step is to
> > > share
> > > > >> the
> > > > >> > > page links with Mariana Tek for QA. Since the project appears
> > to
> > > be
> > > > >> > > private, could you please publish it and provide us with the
> > > > website
> > > > >> > link?
> > > > >> > >
> > > > >> > > Best regards,
> > > > >> > >
> > > > >> > > *Rahul*
> > > > >> > > *Integrations*
> > > > >> > > [image: ipstudio sig block logo-9.png]
> > > > >> > > ipstudio.co
> > > > >> > >
> > > > >> > >
> > > > >> > > On Thu, Sep 18, 2025 at 2:07 AM awaisqazi ***@***.***>
> > > > >> > > wrote:
> > > > >> > >
> > > > >> > >> Merged #1 <
> > > https://github.com/awaisqazi/latinasweatproject/pull/1>
> > > >
> > > > >> > into
> > > > >> > >> main.
> > > > >> > >>
> > > > >> > >> —
> > > > >> > >> Reply to this email directly, view it on GitHub
> > > > >> > >> <
> > > > >> >
> > > > >>
> > > >
> > https://github.com/awaisqazi/latinasweatproject/pull/1#event-19759367807>,
>
> >
> > >
> > > >
> > > > >>
> > > > >> >
> > > > >> > >> or unsubscribe
> > > > >> > >> <
> > > > >> >
> > > > >>
> > > >
> > >
> >
> https://github.com/notifications/unsubscribe-auth/BV3ZBAP6E7Q5RK7LB3T3C333THBBJAVCNFSM6AAAAACGZAN3SWVHI2DSMVQWIX3LMV45UABCJFZXG5LFIV3GK3TUJZXXI2LGNFRWC5DJN5XDWMJZG42TSMZWG44DANY>
>
> >
> > >
> > > >
> > > > >>
> > > > >> >
> > > > >> > >> .
> > > > >> > >> You are receiving this because you authored the
> thread.Message
> > > ID:
> > > > >> > >> ***@***.***>
> > > > >> > >>
> > > > >> > >
> > > > >> >
> > > > >> > —
> > > > >> > Reply to this email directly, view it on GitHub
> > > > >> > <
> > > > >>
> > > >
> > >
> >
> https://github.com/awaisqazi/latinasweatproject/pull/1#issuecomment-3304790565>,
>
> >
> > >
> > > >
> > > > >>
> > > > >> > or unsubscribe
> > > > >> > <
> > > > >>
> > > >
> > >
> >
> https://github.com/notifications/unsubscribe-auth/AELHOAK4SN66MLRBWPMOPA33THRULAVCNFSM6AAAAACGZAN3SWVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZTGMBUG44TANJWGU>
>
> >
> > >
> > > >
> > > > >>
> > > > >> > .
> > > > >> > You are receiving this because you modified the open/close
> > > > >> state.Message
> > > > >> > ID: ***@***.***>
> > > > >> >
> > > > >>
> > > > >> —
> > > > >> Reply to this email directly, view it on GitHub
> > > > >> <
> > > >
> > >
> >
> https://github.com/awaisqazi/latinasweatproject/pull/1#issuecomment-3304793578>,
>
> >
> > >
> > > >
> > > > >> or unsubscribe
> > > > >> <
> > > >
> > >
> >
> https://github.com/notifications/unsubscribe-auth/BV3ZBAPYV4XWOZUUGYO3ZTD3THR3PAVCNFSM6AAAAACGZAN3SWVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZTGMBUG44TGNJXHA>
>
> >
> > >
> > > >
> > > > >> .
> > > > >> You are receiving this because you authored the thread.Message
> ID:
> > > > >> ***@***.***>
> > > > >>
> > > > >
> > > >
> > > > —
> > > > Reply to this email directly, view it on GitHub
> > > > <
> > >
> >
> https://github.com/awaisqazi/latinasweatproject/pull/1#issuecomment-3308622726>,
>
> >
> > >
> > > > or unsubscribe
> > > > <
> > >
> >
> https://github.com/notifications/unsubscribe-auth/AELHOAMJYGK2REBLWICFBPL3TLRWDAVCNFSM6AAAAACGZAN3SWVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZTGMBYGYZDENZSGY>
>
> >
> > >
> > > > .
> > > > You are receiving this because you modified the open/close
> > state.Message
> > > > ID: ***@***.***>
> > > >
> > >
> > > —
> > > Reply to this email directly, view it on GitHub
> > > <
> >
> https://github.com/awaisqazi/latinasweatproject/pull/1#issuecomment-3309744386>,
>
> >
> > > or unsubscribe
> > > <
> >
> https://github.com/notifications/unsubscribe-auth/BV3ZBANETNJWGBAINUXBEAT3TMN6TAVCNFSM6AAAAACGZAN3SWVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZTGMBZG42DIMZYGY>
>
> >
> > > .
> > > You are receiving this because you authored the thread.Message ID:
> > > ***@***.***>
> > >
> >
> > —
> > Reply to this email directly, view it on GitHub
> > <
> https://github.com/awaisqazi/latinasweatproject/pull/1#issuecomment-3309763966>,
>
> > or unsubscribe
> > <
> https://github.com/notifications/unsubscribe-auth/AELHOALI7Z2JCYDIIVEVUSD3TMO6BAVCNFSM6AAAAACGZAN3SWVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZTGMBZG43DGOJWGY>
>
> > .
> > You are receiving this because you modified the open/close state.Message
> > ID: ***@***.***>
> >
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/awaisqazi/latinasweatproject/pull/1#issuecomment-3309785190>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BV3ZBALAXSZ5DOQ7C2XBPTD3TMQCFAVCNFSM6AAAAACGZAN3SWVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZTGMBZG44DKMJZGA>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
